data-raw/gtsummary_tests.csv

```diff
@@ -1,53 +1,53 @@
-tbl_survfit,TRUE,FALSE,,survival,coxph_lrt,survival::coxph,"\(...) gtsummary:::add_p_tbl_survfit_coxph(..., test_type = ""log"")",TRUE,"survival::coxph(Surv(.) ~ variable, data, ...)",Cox regression (LRT),
-tbl_survfit,TRUE,FALSE,,survival,coxph_wald,,"\(...) gtsummary:::add_p_tbl_survfit_coxph(..., test_type = ""wald"")",TRUE,"survival::coxph(Surv(.) ~ variable, data, ...)",Cox regression (Wald),
-tbl_survfit,TRUE,FALSE,,survival,coxph_score,,"\(...) gtsummary:::add_p_tbl_survfit_coxph(..., test_type = ""sc"")",TRUE,"survival::coxph(Surv(.) ~ variable, data, ...)",Cox regression (Score),
-tbl_continuous,TRUE,FALSE,,stats,anova_2way,,gtsummary:::add_p_test_anova_2way,FALSE,lm(continuous_variable ~ by + variable) %>% broom::glance(),Two-way ANOVA,
-tbl_continuous,TRUE,TRUE,,stats,t.test,stats::t.test,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""t.test"")",TRUE,"t.test(continuous_variable ~ variable, data = data, conf.level = 0.95, ...)",t-test,
-tbl_continuous,TRUE,FALSE,,stats,aov,stats::aov,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""aov"")",FALSE,"aov(continuous_variable ~ variable, data = data) %>% summary()",One-way ANOVA,
-tbl_continuous,TRUE,FALSE,,stats,oneway.test,stats::onewway.test,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""oneway.test"")",TRUE,"oneway.test(continuous_variable ~ variable, data = data)",One-way ANOVA,
-tbl_continuous,TRUE,FALSE,,stats,kruskal.test,stats::kruskal.test,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""kruskal.test"")",FALSE,"kruskal.test(x = data[[continuous_variable]], g = data[[variable]])",Kruskal-Wallis test,
-tbl_continuous,TRUE,FALSE,,stats,wilcox.test,stats::wilcox.test,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""wilcox.test"")",TRUE,"wilcox.test(continuous_variable ~ variable, data = data, ...)",Wilcoxon rank-sum test,
-tbl_continuous,TRUE,FALSE,,lme4,lme4,lme4::glmer,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""lme4"")",FALSE,"lme4::glmer(by ~ (1 \UFF5C group), data, family = binomial) %>% anova(lme4::glmer(variable ~ continuous_variable + (1 \UFF5C group), data, family = binomial))",random intercept logistic regression,
-tbl_continuous,TRUE,FALSE,,stats,ancova,,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""ancova"")",FALSE,lm(continuous_variable ~ variable + adj.vars),ANCOVA,
-tbl_continuous,FALSE,FALSE,,stats,ancova_lme4,,"\(...)  gtsummary:::add_p_test_tbl_summary_to_tbl_continuous(..., test_name = ""ancova_lme4"")",FALSE,"lme4::lmer(continuous_variable ~ variable + adj.vars + (1 \UFF5C group), data)",ANCOVA with random intercept,
\ No newline at end of file
+class,add_p,add_difference,diff_statistic,test_package,test_name,test_fun,fun_to_run,accept_dots,pseudo_code,description,details
```

Comment:
Here there are no differences in content (it seems to me). I guess github diff does not work correctly on .csv

Could it make sense to make it a .txt (even if read and used as csv) to have the correct diff?

Reply:
Usually, it would only show the difference for a CSV file. It looks like the only difference is an extra line break on the last row, which for whatever reason, is appearing as if every line wac changed.

Reply:
ok then nothing to be done! :)